### PR TITLE
Fix test syntax issues passing

### PR DESCRIPTION
**Pull request by lldelisle**
See #530

### CODE REVIEW

**Comment by lldelisle** [comment]
Same question as #533 , I don't know if these changes require a bump version.

**Comment by mvdbeek** [comment]
No, if we're only changing the tests I don't think we need to do that. It doesn't affect the workflow itself.

File: workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml [hunk 1]
```diff
@@ -12,22 +12,18 @@
     Ploidy: 1
   outputs:
     GenomeScope linear plot:
-      class: File
```
Comment (mvdbeek) [inline workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml:15]: @jmchilton do we like this ? It's a great union discriminator that would give us cleaner validation reports, I think ?

**Review by mvdbeek — APPROVED** [review]

File: workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml (hunk 1 above)
Comment (jmchilton) [reply workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml:15]: The inputs you could argue are files. These are datasets - definitely not files. But the way CWL uses the word ``File`` is also just a real problem for my literal brain. We've already given into the syntax so why not just continue I guess. It is hard to appreciate consistency and technical quality of something that is wrong 😆. We can make this required in the best practice schema I think 😿. Will you give me the ``that`` syntax on the same argument about having a good clean discriminator 😆?

Comment (mvdbeek) [reply workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml:15]: Sure, that's fine. I guess we're still at the stage of defining what works, and then next step consolidating into something we like ?

Comment (jmchilton) [reply workflows/VGP-assembly-v2/kmer-profiling-hifi-VGP1/kmer-profiling-hifi-VGP1-tests.yml:15]: https://github.com/galaxyproject/galaxy/pull/18889

**Comment by mvdbeek** [comment]
Attention: deployment failure!

https://github.com/galaxyproject/iwc/actions/runs/11107437308

**Pull request merged**